com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.RenderGraph.cs

```diff
@@ -164,16 +164,29 @@ class CustomPostProcessData
             public CustomPostProcessVolumeComponent customPostProcess;
         }
 
-        TextureHandle GetPostprocessOutputHandle(RenderGraph renderGraph, string name)
+        TextureHandle GetPostprocessOutputHandle(TextureHandle previousTexture, RenderGraph renderGraph, string name)
```

Comment:
I am not so sure on this. Both this and the GetPostprocessUpsampledOutputHandle are identical if not for the hardware dynamic res is enabled. 

I think I'd rather have that being an explicit parameter rather than relying on the source. 

So something like 


GetPostprocessOutputHandle(RenderGraph renderGraph, string name, bool enableHWDynamicRes) 

and then caller can be explicit like 

TextureHandle dest = GetPostprocessOutputHandle(renderGraph, "Motion Blur Destination", enableHWDynamicRes: true);

and 

                    TextureHandle dest = GetPostprocessOutputHandle(renderGraph, "Contrast Adaptive Sharpen Destination", enableHWDynamicRes: true);

Reply:
We can still have GetPostprocessUpsampledOutputHandle as a verbose wrapper around  GetPostprocessOutputHandle(renderGraph, name, enableHWDynamicRes: false)

Reply:
done!